PWGHF/treeHF/AliHFJetFinder.cxx

```diff
@@ -465,8 +465,8 @@ Bool_t AliHFJetFinder::CheckTrack(AliAODTrack *track) {
 //Apply filter bit selection on tracks
 Bool_t AliHFJetFinder::CheckFilterBits(AliAODTrack *track) {
 
-  if(track->TestBits(BIT(4))/TMath::Power(2,4)!=1 ) return false;
-  if(!track->IsHybridGlobalConstrainedGlobal()) return false;
+  if((track->TestBits(BIT(4))/TMath::Power(2,4)!=1) && (track->TestBits(BIT(9))/TMath::Power(2,9)!=1)) return false;
```

Comment:
```suggestion
  if((track->TestBits(BIT(4)) == 0) && (track->TestBits(BIT(9)) == 0)) return false;
```
TMath::Power is a (heavy) floating point operation that is really not needed here.